domain/controllernode/state/state_test.go

```diff
@@ -44,14 +44,23 @@ func (s *stateSuite) SetUpTest(c *tc.C) {
 	s.state = NewState(s.TxnRunnerFactory())
 }
 
-func (s *stateSuite) TestCurateNodes(c *tc.C) {
+func (s *stateSuite) TestAddDqliteNode(c *tc.C) {
 	db := s.DB()
 
+	// This
```

Comment:
`& that`

Reply:
I clearly had something important to say :laughing: